client/packages/system/src/IndicatorsDemographics/DetailView/IndicatorsDemographics.tsx

```diff
@@ -225,22 +225,19 @@ const IndicatorsDemographicsComponent = () => {
 
   return (
     <>
-      <AppBarButtons
-        createNewRow={createNewRow}
-        rows={Object.values(draft)}
-      ></AppBarButtons>{' '}
+      <AppBarButtons createNewRow={createNewRow} rows={Object.values(draft)} />
       <Box sx={{ width: '100%' }} padding={0}>
         <GrowthRow
           columns={columns}
           data={headerDraft}
           isLoading={isLoadingProjection}
           setData={handleGrowthChange}
-        ></GrowthRow>
+        />
         <DataTable
           data={Object.values(draft)}
           columns={columns}
           id={'indicators-demographics-table'}
-        ></DataTable>
+        />
```

Comment:
Unrelated, but spotted when testing out this area -- self-closing tags are much more succinct and clear when no inner content :)